### PR TITLE
language_lilypond

### PR DESCRIPTION
**Pull request by jperon**
This is my first attempt, so feel free to amend!

### CODE REVIEW

File: language_lilypond.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "LilyPond",
+  files = { "%.ly$" },
```
Comment (Guldoman) [inline language_lilypond.lua:6]: Seems like another possible extension is `.ily`, so you could do:
```suggestion
  files = { "%.i?ly$" },
```

**Review by Guldoman — COMMENTED** [review]
Thank you for your contribution!

You should also update the `manifest.json` file in the root of this repository.

Seems like `%{` and `%}` are used for block comments, so that should probably be added.

File: plugins/language_lilypond.lua [hunk 2]
```diff
@@ -0,0 +1,21 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "LilyPond",
+  files = { "%.i?ly$" },
+  comment = "%%",
```
Comment (Guldoman) [inline plugins/language_lilypond.lua:7]:
```suggestion
  comment = "%%",
  block_comment = { "%%{", "%%}" },
```
This enables the command to add block comments.

**Comment by jperon** [comment]
Should be ready now.

**Review by Guldoman — APPROVED** [review]
I'll wait to merge this for a bit, to allow you to add new patterns if anything comes up, and to allow others to comment.

**Pull request merged**